engine/apps/mobile_app/models.py

```diff
@@ -51,9 +51,7 @@ def create_auth_token(cls, user: User, organization: Organization) -> Tuple["Mob
 
 
 class MobileAppAuthToken(BaseAuthToken):
-    user = models.ForeignKey(
-        to=User, null=False, blank=False, related_name="mobile_app_auth_tokens", on_delete=models.CASCADE
-    )
+    user = models.OneToOneField(to=User, null=False, blank=False, on_delete=models.CASCADE)
```

Comment:
main focus of the PR. Rest is related to new API endpoint tests I added.

Reply:
do we have one to many relations already?
Do we need any data migration/cleanup?

Reply:
I've checked all of our environments and we're all good, no migration needed.

We only had one instance of this pop up (which affected @imtoori), this was manually fixed.

Reply:
What about OSS instances, I guess it's possible that some duplicates are created on some of the OSS stacks?

Reply:
see [comment above](https://github.com/grafana/oncall/pull/1482/files#r1129372892)